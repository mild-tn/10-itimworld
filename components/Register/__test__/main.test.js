/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import MainRegister from '../Main'

describe('<MainRegister />', () => {
  it('MainRegister should be renders', () => {
    const component = renderer.create(<MainRegister />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
