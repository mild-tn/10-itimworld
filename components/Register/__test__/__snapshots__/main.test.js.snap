// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainRegister /> MainRegister should be renders 1`] = `
<div>
  register component
</div>
`;
